packages/js/src/plugins/auctionHouseModule/operations/depositToBuyerAccount.ts

```diff
@@ -0,0 +1,255 @@
+import { ConfirmOptions, PublicKey } from '@solana/web3.js';
+import type { Metaplex } from '@/Metaplex';
+import { TransactionBuilder } from '@/utils';
+import {
+  createAuctioneerDepositInstruction,
+  createDepositInstruction,
+  DepositInstructionAccounts,
+} from '@metaplex-foundation/mpl-auction-house';
+import {
+  useOperation,
+  Operation,
+  OperationHandler,
+  Signer,
+  toPublicKey,
+  isSigner,
+  SolAmount,
+  SplTokenAmount,
+} from '@/types';
+import { SendAndConfirmTransactionResponse } from '../../rpcModule';
+import { findAssociatedTokenAccountPda } from '../../tokenModule';
+import { AuctionHouse } from '../models';
+import { findAuctioneerPda, findAuctionHouseBuyerEscrowPda } from '../pdas';
+import { AuctioneerAuthorityRequiredError } from '../errors';
+
+// -----------------
+// Operation
+// -----------------
+
+const Key = 'DepositToBuyerAccountOperation' as const;
+
+/**
+ * Adds funds to the user's buyer escrow account for the given auction house.
+ *
+ * ```ts
+ * await metaplex
+ *   .auctionHouse()
+ *   .depositToBuyerAccount({ auctionHouse, buyer, amount })
+ *   .run();
+ * ```
+ *
+ * @group Operations
+ * @category Constructors
+ */
+export const depositToBuyerAccountOperation =
+  useOperation<DepositToBuyerAccountOperation>(Key);
+
+/**
+ * @group Operations
+ * @category Types
+ */
+export type DepositToBuyerAccountOperation = Operation<
+  typeof Key,
+  DepositToBuyerAccountInput,
+  DepositToBuyerAccountOutput
+>;
+
+/**
+ * @group Operations
+ * @category Inputs
+ */
+export type DepositToBuyerAccountInput = {
+  /** The Auction House in which escrow buyer deposits funds. */
+  auctionHouse: Pick<
+    AuctionHouse,
+    | 'address'
+    | 'authorityAddress'
+    | 'hasAuctioneer'
+    | 'isNative'
+    | 'treasuryMint'
+    | 'feeAccountAddress'
+  >;
+  /**
+   * The buyer who deposits funds.
+   * This expects a Signer.
+   *
+   * @defaultValue `metaplex.identity()`
+   */
+  buyer?: Signer;
+
+  /**
+   * The Auction House authority.
+   *
+   * @defaultValue `auctionHouse.authority`
+   */
+  authority?: PublicKey;
```

Comment:
If this no longer needs to be a `Signer`, can we not just use `auctionHouse.authorityAddress` directly since it's required in the `Pick` type above?